### ISSUE TITLE
sportowefakty.wp.pl

### ISSUE THREAD

**Issue by NanoMeow** [NanoMeow/QuickReports#1760]
### Basic Information

**Test link:** `https://sportowefakty.wp.pl/`
**Category:** anti-adblock
**Reported from:** Poland
**Template version:** 2

### User Environment

**Browser:** Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/76.0.3809.110 Safari/537.36 Vivaldi/2.7.1628.26
**Extension:** Nano Defender 15.0.0.165

### Additional Message

Original URL: `https://sportowefakty.wp.pl/zuzel/ps-poznan-stowarzyszenie`

**Comment by krystian3w** [comment]
Enable `AdGuard Base List` as long uAsset no override:

`#@#+js(set-constant, Object.prototype.bodies, true)`
`#@#+js(abort-on-property-read, WP.inline)`

**Comment by jspenguin2017** [comment]
See: https://github.com/NanoMeow/QuickReports/issues/1741

**Comment by okiehsch** [comment]
> Enable AdGuard Base List as long uAsset no override:
#@#+js(set-constant, Object.prototype.bodies, true)
#@#+js(abort-on-property-read, WP.inline)

In my opinion the polish list should remove those filters or keep them and add the `AdGuard Base List` filters.

**Comment by krystian3w** [comment]
So Poland must copy all 27k filters? OMG...

I suppose AdGurad no remove ads with procedural filtering, to disable:
```
#@#+js(set-constant, Object.prototype.bodies, true)
#@#+js(abort-on-property-read, WP.inline)
```

First exsist in similar syntax (get prototype used) in AG base.

**Comment by okiehsch** [comment]
No, they should remove their
````
#@#+js(set-constant, Object.prototype.bodies, true)
#@#+js(abort-on-property-read, WP.inline)
````
filters because it triggers the anti adblock warning.

**Comment by krystian3w** [comment]
but comeback 99% one day ads - no better try allow 2-3 google ads files and maybe two EasyList bait cosmetic filters?

AdGuard fix in this method without disable scriplet from MajkiIT repo.
Olegwukr newer write any post in topic why no added all propose solutions.

**Comment by okiehsch** [comment]
Well, right now the only thing  users of the polish list see is an anti adblock message.

I am pretty sure they would prefer a solution that would let them actually use the site.

**Comment by krystian3w** [comment]
No, is not useable with unlocked ads. Too much in XXI centaury.

@gorhill possible for Poland enable AG base in uBO 1.22.x or 1.23?

**Comment by okiehsch** [comment]
If the site works for you if you enable AdGuard base why not adding the relevant filters of that list to the polish list?

**Comment by krystian3w** [comment]
Because nobody follow AdamWr to AdGuard Base comitts and logger uBo may no show all excptions (example fast bait on naekranie.pl no visible in uBo logger or need very luck).

**Comment by krystian3w** [comment]
And hard working with Olegwukr if no tell nothing more as commits with copied 2 form 6 filters.

**Comment by NanoMeow** [comment]
### Basic Information

**Test link:** `https://sportowefakty.wp.pl/boks-zawodowy/841075/boks-watpliwosci-ws-dawida-kosteckiego-opinia-bieglego-obala-wersje-romana-giert`
**Category:** anti-adblock
**Reported from:** Poland
**Template version:** 2

### User Environment

**Browser:** Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/76.0.3809.110 Safari/537.36 Vivaldi/2.7.1628.30
**Extension:** Nano Defender 15.0.0.166

### Additional Message

**Comment by NanoMeow** [comment]
### Basic Information

**Test link:** `https://sportowefakty.wp.pl/zuzel`
**Category:** anti-adblock
**Reported from:** Poland
**Template version:** 2

### User Environment

**Browser:** Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/76.0.3809.110 Safari/537.36 Vivaldi/2.7.1628.26
**Extension:** Nano Defender 15.0.0.166

### Additional Message

**Comment by okiehsch** [comment]
@MajkiIT

For the anti-adblock messages at 
`o2.pl,pudelek.pl,wp.pl`
caused by your filters
adding
````
@@||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$domain=o2.pl|pudelek.pl|wp.pl
wp.pl#@#body > div:nth-of-type(1)[style^="display: block;"]:not(#page)
o2.pl,pudelek.pl,wp.pl#@#.pub_300x250
o2.pl,pudelek.pl,wp.pl#@#.text_ads
````
fixes the issue on my end.

**Comment by MajkiIT** [comment]
@okiehsch Unfortinatly not working :(

**Comment by krystian3w** [comment]
WP something changed now no help also `AG base List`.

**Comment by krystian3w** [comment]
I see bug who merege 2 lines into 1:

![obraz](https://user-images.githubusercontent.com/35370833/64200597-218b5200-ce8d-11e9-8f17-bbd216a1feb8.png) |
--- |

this is two needed scriplets.

**Comment by NanoMeow** [comment]
### Basic Information

**Test link:** `https://sportowefakty.wp.pl/zuzel`
**Category:** anti-adblock
**Reported from:** Poland
**Template version:** 2

### User Environment

**Browser:** Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/77.0.3865.62 Safari/537.36 Vivaldi/2.8.1657.8
**Extension:** Nano Defender 15.0.0.168

### Additional Message

**Comment by okiehsch** [comment]
> Well, right now the only thing users of the polish list see is an anti adblock message.
I am pretty sure they would prefer a solution that would let them actually use the site.

This is still the case three weeks later, so I have disabled those filters in `uBO-filters`.

**Comment by krystian3w** [comment]
So also update heavy ads (random class, URL, position in DOM).

**Comment by okiehsch** [comment]
I don't see any ads with the polish list enabled on my end.

**Comment by krystian3w** [comment]
On homepage:

![obraz](https://user-images.githubusercontent.com/35370833/65348099-ae9a0f00-dbe0-11e9-84e8-1992b8b82878.png)

![obraz](https://user-images.githubusercontent.com/35370833/65348143-c96c8380-dbe0-11e9-8f81-51d3f207c9a0.png)


Placeholders

**Comment by krystian3w** [comment]
Exceptions die again detect uBO:

```
https://finanse.wp.pl/jadlam-ze-smietnikow-nie-uwierzycie-ile-tam-skarbow-6426560515410049a
```

**Comment by okiehsch** [comment]
````
@@||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$script,domain=wp.pl
wp.pl#@#body > div:nth-of-type(1)[style^="display: block;"]:not(#page)
pl#@#.ads
````

**Comment by NanoMeow** [comment]
### Basic Information

**Test link:** `https://sportowefakty.wp.pl`
**Category:** anti-adblock
**Reported from:** Poland
**Template version:** 2

### User Environment

**Browser:** Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:70.0) Gecko/20100101 Firefox/70.0
**Extension:** Nano Defender 15.0.0.170

### Additional Message

Original URL: `https://sportowefakty.wp.pl/skoki-narciarskie/855637/ps-w-skokach-kosmiczny-lot-jana-hoerla-na-bergisel-skoczek-coraz-smielej-daje-o-`

**Comment by krystian3w** [comment]
Unlock read or block placeholders...

WP follow moddification in Polish Repository...

**Comment by NanoMeow** [comment]
### Basic Information

**Test link:** `https://sportowefakty.wp.pl/`
**Category:** anti-adblock
**Reported from:** Poland
**Template version:** 2

### User Environment

**Browser:** Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/77.0.3865.62 Safari/537.36 Vivaldi/2.8.1657.8
**Extension:** Nano Defender 15.0.0.170

### Additional Message

**Comment by lain566** [comment]
I don't get anti adblock warning, but the site is broken if I enable Polish regional filter lists.

**Comment by krystian3w** [comment]
Yeah WP update in few hours `Object.prototype` varible after commit on Polish repository.

Bad:

`portal.abczdrowie.pl,parenting.pl,kafeteria.pl,dobreprogramy.pl,o2.pl,pudelek.pl,wp.pl#@#+js(set-constant.js, Object.prototype.rekid, undefined)`

https://github.com/AdguardTeam/AdguardFilters/issues/44613#issuecomment-559718849


Latest is:

```
~www.wp.pl,gadzetomania.pl,komorkomania.pl,portal.abczdrowie.pl,parenting.pl,dobreprogramy.pl,o2.pl,pudelek.pl,wp.pl#%#//scriptlet("set-constant", "Object.prototype.rebuild", "undefined")
```


The Polish repository currently does not have a good JavaScript specialist to update this variable every 2-4 days.

Apparently, if you don't copy from AdGuard, the script filter for AdGuard users works even for 2 weeks

**Comment by krystian3w** [comment]
When removing a message from the cosmetic procedure filter:

- `:has()`,
- `:has-text()`, 
- `:matches-css...()`,
- `:xpath()`) 

they also know how to block pop-up removal:
- change images,
-  CSS values, 
- text is created pseudo elements with invisible characters.

**Comment by lain566** [comment]
@krystian3w I mean, for example ....

`https://sportowefakty.wp.pl/sportowybar/wideo/36279/dziejesiewsporcie-sagan-dal-autograf-w-trakcie-wyscigu`

If you go to this page, with the Polish filter lists enabled, the player will not load. It seems that someone thought it was a good idea to hide the cookie notice.

**Comment by krystian3w** [comment]
Cookies no possible hide, accept or not loaded video:

https://github.com/PolishFiltersTeam/PolishCookieConsent

**Comment by lain566** [comment]
Anyway, there are more broken things on the page, this was one of them.

**Comment by krystian3w** [comment]
Again you broke header?

**Comment by lain566** [comment]
As I said before

> the site is broken if I enable Polish regional filter lists.

**Comment by krystian3w** [comment]
unsubrcibe please.

**Comment by lain566** [comment]
> unsubrcibe please.

I only subscribed to be able to fix anti adblock, the breakages of the site that I mentioned before, is for you to know, and can fix it, I'm sorry if my words sound hard, and I understand, everyone is wrong, nobody is perfect.

**Comment by jspenguin2017** [comment]
This issue is sticky and we are not getting anywhere in term of finding a durable solution. The conversation is becoming unconstructive, locking.

**Comment by NanoMeow** [comment]
### Basic Information

**Test link:** `https://sportowefakty.wp.pl/pilka-nozna/860558/adam-buksa-w-szatni-new-england-revolution-z-jezykiem-nie-ma-zadnego-problemu?src01=6a4c8&fbclid=IwAR2KoeKEWjwsidQdD-7cPl4Pktxg2bGDHOgoaFvM-TeZX2ndQGQpHgZaYSg`
**Category:** anti-adblock
**Reported from:** Poland
**Template version:** 2

### User Environment

**Browser:** Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/79.0.3945.79 Safari/537.36
**Extension:** Nano Defender 15.0.0.170

### Additional Message

**Comment by NanoMeow** [comment]
### Basic Information

**Test link:** `https://sportowefakty.wp.pl/pilka-nozna/860370/grzegorz-rasiak-pilka-ma-krotka-pamiec?utm_campaign=sgf&utm_medium=sf&utm_source=fb`
**Category:** anti-adblock
**Reported from:** Poland
**Template version:** 2

### User Environment

**Browser:** Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/79.0.3945.79 Safari/537.36
**Extension:** Nano Defender 15.0.0.170

### Additional Message
